src/galaxy/config.py

```diff
@@ -1,13 +1,83 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
+# Copyright (C) 2021 Humanitarian OpenStreetmap Team
+
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+# Humanitarian OpenStreetmap Team
+# 1100 13th Street NW Suite 800 Washington, D.C. 20005
+# <info@hotosm.org>
 
 from configparser import ConfigParser
+import logging
 
 CONFIG_FILE_PATH = "src/config.txt"
 
 config = ConfigParser()
 config.read(CONFIG_FILE_PATH)
 
+log_level = config.get("API_CONFIG", "log_level",fallback=None) # get log level from config
+
+if log_level is None or log_level == 'debug' or log_level == 'DEBUG':  # default will go to debug
+    level=logging.DEBUG 
+elif log_level == 'info' or log_level == 'INFO' :
```

Comment:
Here's a safer option that is completely case insensitive:

`if log_level.lower() == "info": `